Makefile

```diff
@@ -16,6 +16,8 @@ prepare:
 	rm -rf $(BUILD_DIR)
 	mkdir -p $(BUILD_DIR)/SPECS $(BUILD_DIR)/SOURCES
 	cp $(SPEC_FILE) $(BUILD_DIR)/SPECS/
+	# rpmbuild may get confused on multiple spec files in tarball
+	find . -mindepth 2 -name '*.spec' -delete
```

Comment:
Vendored dependencies may introduce additional spec files, if respective repos have ones. The rpmbuild command will pick tem all and complain about duplicate sections. Therefore, at preparation stage we delete all spec files, except the one in root folder of tarball.